src/main/java/game/applicationBase/LocalFileTool.java

```diff
@@ -0,0 +1,137 @@
+
+        SceneStore sceneStore = gson.fromJson(sceneStoreReader, SceneStore.class);
+        ItemStore itemStore = gson.fromJson(itemStoreReader, ItemStore.class);
+        ActionStore actionStore = gson.fromJson(actionStoreReader, ActionStore.class);
+
+        Game game = new Game(sceneStore.toIntegerSceneMap(),
+                actionStore.toIntegerActionMap(), itemStore.toStringItemMap(), sceneStore.getStartSceneId());
+
+        return game;
+    }
+
+    public static Game makeGameFromSaveFile(ClassLoader classLoader, File file) throws FileNotFoundException {
+        setupDirectories(classLoader);
+
+        if (file.exists() && file.isFile()) {
+            JsonReader jsonReader = new JsonReader(new FileReader(file));
+            return new Gson().fromJson(jsonReader, Game.class);
+        } else {
+            throw new FileNotFoundException("Could not find save file.");
+        }
+    }
+
+    private static void setupDirectories(ClassLoader classLoader) {
+        File mainDir = new File(MAIN_DIR);
+        File gameDir = new File(GAME_DIR);
+        File saveDir = new File(SAVE_DIR);
+
+        if (!mainDir.exists()) {
+            mainDir.mkdir();
+        }
+
+        if (!gameDir.exists()) {
+            gameDir.mkdir();
+        }
+
+        if (!saveDir.exists()) {
+            saveDir.mkdir();
+        }
+
+        if (!mainItemFile.exists()) {
+            URL resource = classLoader.getResource("main-game.items.json");
+            if (resource == null) {
+                throw new IllegalArgumentException("Resource not found!");
+            } else {
+                File resourceFile = new File(resource.getFile());
+                try {
+                    FileUtils.copyFile(resourceFile, mainItemFile);
+                } catch (IOException e) {
+                    e.printStackTrace();
+                }
+            }
+        }
+
+        if (!mainSceneFile.exists()) {
+            URL resource = classLoader.getResource("main-game.scenes.json");
+            if (resource == null) {
+                throw new IllegalArgumentException("Resource not found!");
+            } else {
+                File resourceFile = new File(resource.getFile());
+                try {
+                    FileUtils.copyFile(resourceFile, mainSceneFile);
+                } catch (IOException e) {
+                    e.printStackTrace();
+                }
+            }
+        }
+
+        if (!mainActionsFile.exists()) {
+            URL resource = classLoader.getResource("main-game.actions.json");
+            if (resource == null) {
+                throw new IllegalArgumentException("Resource not found!");
+            } else {
+                File resourceFile = new File(resource.getFile());
+                try {
+                    FileUtils.copyFile(resourceFile, mainActionsFile);
+                } catch (IOException e) {
+                    e.printStackTrace();
+                }
+            }
+        }
+    }
+
+}
```

Comment:
Very organized and clean implementation of file management and directory operations.